.travis.yml

```diff
@@ -1,10 +1,17 @@
+sudo: required
+dist: trusty
+
 language: go
 
 go:
-  - 1.6
+- 1.6.2
+before_install:
+- sudo apt-get -qq update
+- sudo apt-get install -y libvirt-dev qemu-kvm libvirt-bin
+- sudo usermod -a -G libvirtd $USER
 # We shouldn't need to install anything because we use vendoring.
 install:
 - echo "Don't run anything."
 
 # Run unit tests and linters.
-script: make test
+script: sudo -E su $USER -c "env PATH=$PATH make test"
```

Comment:
The reason for sudo here is because the user has only just been added to the libvirtd group, required to use the libvirt socket as non-root user, so sudo is effectively being used here to switch to the current user (hence the `su $USER`) but pick up the new group membership. Funky workaround :)

So it's not running as root, but effectively as the same user in a new shell with required additional group membership.